public/javascripts/autofill-profile-form.js

```diff
@@ -17,3 +17,10 @@ function checkDate(date) {
     date.value = "";
   }
 }
+
+function removeTextAreaWhiteSpace() {
+  const myTxtArea = document.getElementById(`bio`);
+  myTxtArea.value = myTxtArea.value.replace(/^\s*|\s*$/g, ``);
```

Comment:
Use JS inbuilt method if this is what you want to achieve https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim

Reply:
Nice. Honestly, had no clue it existed, is pretty useful.